src/request/plan.rs

```diff
@@ -345,6 +345,18 @@ pub(crate) async fn handle_region_error<PdC: PdClient>(
     }
 }
 
+pub(crate) async fn handle_rpc_error<PdC: PdClient>(
```

Comment:
I think it would be better to name as `invalidate_cache` for wider usage.

It would be nice if you can use this method for `handle_region_error` & `handle_other_error` to eliminate duplicated codes. But it is still OK if you do not want to change them in this PR.